SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -116,7 +117,6 @@ for FUNCTION in FAIL_DELAY               \
                 CRACKLIB_DICTPATH        \
                 PASS_CHANGE_TRIES        \
                 PASS_ALWAYS_WARN         \
-                CHFN_AUTH ENCRYPT_METHOD \
```

Comment:
Could you add a comment here, or in the change long, explaining why we don't want to run this for `CHFN_AUTH`  or `ENCRYPT_METHOD` anymore?